lexicons/com/atproto/server/getServiceAuth.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.getServiceAuth",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a signed token on behalf of the requesting DID for the requested service.",
+      "parameters": {
+        "type": "params",
+        "required": ["aud"],
+        "properties": {
+          "aud": {
+            "type": "string",
+            "format": "did",
```

Comment:
if we make this `format: uri` it would work with DIDs and leave the door open to other audience URIs/values... but probably best to keep it DID-only.

Reply:
I like just leaving it DID for now until we see the need for that. I think we can upgrade into `format: uri` without much trouble